plugins/modules/zabbix_authentication.py

```diff
@@ -0,0 +1,584 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright: (c) 2022, ONODERA Masaru <masaru-onodera@ieee.org>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = '''
+---
+module: zabbix_authentication
+
+short_description: Update Zabbix authentication
+
+description:
+   - This module allows you to modify Zabbix authentication setting.
+
+author:
+    - ONODERA Masaru(@masa-orca)
+
+requirements:
+    - "zabbix-api >= 0.5.4"
+
```

Comment:
can you add this also please?

```suggestion
version_added: 1.6.0
```

Reply:
Thanks! I add it.